docker/postgres-kanister-tools/Dockerfile

```diff
@@ -4,15 +4,15 @@ ARG TOOLS_IMAGE
 FROM ${TOOLS_IMAGE} AS TOOLS_IMAGE
 
 # Actual image base
-FROM postgres:16.1-bullseye
+FROM postgres:16-bullseye
 
 ENV DEBIAN_FRONTEND noninteractive
 
 USER root
 
 RUN apt-get update && apt-get -y install curl python3 groff less jq python3-pip && \
-    pip3 install --break-system-packages --upgrade pip && \
-    pip3 install --break-system-packages --upgrade awscli && \
+    pip3 install --upgrade pip && \
+    pip3 install --upgrade awscli && \
```

Comment:
Is it related to K8s dep upgrade? If not, please avoid making changes in the same PR.

Reply:
Understood, this is not coming from my changes but because of merging master changes.

Reply:
Please rebase with master, the PR should not contain other commits. Lets fix that

Reply:
Done